### DISCUSSION TITLE
⚠️ Notice for people making forks

### DISCUSSION THREAD

**Discussion by ppkantorski** [ppkantorski/Ultrahand-Overlay#46] [Announcements]
Just a heads up for people forking.  I have no issues with forks of Ultrahand, but if your fork is public please leave my copyright notices intact (without modifications).  This includes banners with links to my original project and descriptions.  However, community contributions and suggestions to Ultrahand are welcome/encouraged.

But if you would like to assert your own copyright on your fork and modifications, add your copyright notice/banner (ideally with details on your contributions/modifications) before or after mine in source code. 
Appreciate your understanding. 🙏🏼

**Comment by sskyNS** [comment]
[ovlmenu.zip](https://github.com/user-attachments/files/16827365/ovlmenu.zip)
I changed the plugin name and the distance the name draws, including restoring FPS recording and a few other simple fixes.
More info and detils:https://github.com/sskyNS/Ultrahand-Overlay @ppkantorski

**Reply by sskyNS** [reply]
> [ovlmenu.zip](https://github.com/user-attachments/files/16827365/ovlmenu.zip) 我更改了插件名称和名称绘制的距离，包括恢复 FPS 记录和其他一些简单的修复。 更多信息和细节：https://github.com/sskyNS/Ultrahand-Overlay @ppkantorski

New update：Color changes for FPS counts and dynamic colors for titles
check this:https://github.com/sskyNS/Ultrahand-Overlay/commit/d9280a1d3629bc6e47e685a70427f851c052378c
